### ISSUE TITLE
A11Y: Assign article role to individual posts

### ISSUE THREAD

**Issue by schulle4u** [chenasraf/nextcloud-forum#223]
### Feature Summary

Currently, screen reader users find it difficult to navigate between individual posts within a thread because they are not semantically distinguished from one another. Therefore each post should be assigned a landmark or role to improve navigation using a screen reader. 

### Problem or Use Case

When reading a thread, I often want to jump straight to a specific post. As there is no real structure for screen readers, I waste time trying to find the information I need, and I can only search for something using the text search function. If every post had a landmark, navigation would be much quicker.

### Proposed Solution

For this purpose, the article element (recommended) or the article role are suitable. You then can press e.g. the d key in NVDA's browse mode to jump between landmarks. 

### Willing to Contribute?

None
